packages/moonstone/Slider/SliderBehaviorDecorator.js

```diff
@@ -159,6 +159,7 @@ const SliderBehaviorDecorator = hoc(defaultConfig, (config, Wrapped) => {
 
 			return (
 				<Wrapped
+					role="slider"
```

Comment:
Please add `role` after `onFocus`.

Reply:
@ybsung 
Usually the role is above the other prop.
Please see the code below.

https://github.com/enactjs/enact/blob/master/packages/moonstone/internal/SliderDecorator/SliderDecorator.js#L598
https://github.com/enactjs/enact/blob/release-2.0.0-beta.2-test/packages/ui/Button/Button.js#L174
https://github.com/enactjs/enact/blob/release-2.0.0-beta.2-test/packages/ui/Image/Image.js#L167